apps/prairielearn/src/tests/groupExam.test.ts

```diff
@@ -1,7 +1,7 @@
 import { assert } from 'chai';
 import * as cheerio from 'cheerio';
 import fetch from 'node-fetch';
-import fetchCookie = require('fetch-cookie');
+import fetchCookie from 'fetch-cookie';
```

Comment:
From the changelog:

> Breaking: move away from CJS module.exports = fetchCookie style, instead use more standard and future-proof exports.default = fetchCookie. When using CJS, you now need to const fetchCookie = require('fetch-cookie').default instead of const fetchCookie = require('fetch-cookie') previously.